tests/android/webview.spec.ts

```diff
@@ -77,4 +77,22 @@ test('select webview from socketName', async function({ androidDevice }) {
 
   await newPage.close();
   await context.close();
-});
\ No newline at end of file
+});
+
+test('webview.pages', async function({ androidDevice }) {
+  test.slow();
+  // open first page
+  const context = await androidDevice.launchBrowser();
+  // open second page
+  const newPage = await context.newPage();
+  newPage.goto('about:blank');
+
+  const webview = await androidDevice.webView({ socketName: 'webview_devtools_remote_playwright_test' });
```

Comment:
The pages in lines 85 and 87 are created in the browser, those are not web views. I don't think this tests will pass because there are no webviews with the webview_devtools_remote_playwright_test socket name.